packages/odyssey-react-mui/src/addSpecificity.test.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) 2025-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { addSpecificity } from "./addSpecificity.js";
+
+describe("addSpecificity", () => {
+  it("should add specificity with direct selector", () => {
+    expect(addSpecificity(2, ".MuiButton-root")).toBe("&&.MuiButton-root");
+  });
+
+  it("should add specificity with descendant selector", () => {
+    expect(addSpecificity(2, " .MuiButton-root")).toBe("&& .MuiButton-root");
+  });
+
+  it("should work with class references", () => {
+    const buttonClasses = { root: "MuiButton-root" };
+    expect(addSpecificity(2, ".", buttonClasses.root)).toBe(
```

Comment:
This'll make the code less funky (random extra args) and more consistent:

```suggestion
    expect(addSpecificity(2, ".".concat(buttonClasses.root))).toBe(
```